src/Tizen.NUI.Components/Style/ButtonStyle.cs

```diff
@@ -188,7 +188,7 @@ public ButtonStyle(ButtonStyle style) : base(style)
         /// <since_tizen> 8 </since_tizen>
         public Extents IconPadding
         {
-            get => ((Extents)GetValue(IconPaddingProperty)) ?? (iconPadding = new Extents());
+            get => (Extents)GetValue(IconPaddingProperty);
```

Comment:
There are code that assume buttonStyle.IconPadding is not null.
For example, in Button.Internal.cs: 220,
```
            int iconPaddingStart = buttonStyle.IconPadding.Start;
            int iconPaddingEnd = buttonStyle.IconPadding.End;
            int iconPaddingTop = buttonStyle.IconPadding.Top;
            int iconPaddingBottom = buttonStyle.IconPadding.Bottom;
```

In my opinion, they need to be fixed to prevent crash.